RInAppMessaging/Classes/Extensions/CGSize+IAM.swift

```diff
@@ -0,0 +1,6 @@
+extension CGSize {
+    /// returns a `CGSize` with the smallest integer values for its size to prevent drawing on pixel boundaries
+    var integral: CGSize {
```

Comment:
Is that as an adjective? (Checking my English)

Reply:
Used as a noun but I actually pulled this wording from [`CGRect.integral`](https://developer.apple.com/documentation/coregraphics/cgrect/1456348-integral)